tpm2/tpm2.go

```diff
@@ -1187,12 +1187,18 @@ func NVUndefineSpace(rw io.ReadWriter, ownerAuth string, owner, index tpmutil.Ha
 }
 
 // NVUndefineSpaceSpecial This command allows removal of a platform-created NV Index that has TPMA_NV_POLICY_DELETE SET.
```

Comment:
Could you please add a unit test for this function? That will make it easier to help debug what you're seeing if there are still problems.

Reply:
Done